packages/augur-sdk-lite/src/templates/constants.ts

```diff
@@ -57,6 +57,7 @@ export const INDEXES = 'Indexes';
 export const BITCOIN = 'Bitcoin';
 export const ETHEREUM = 'Ethereum';
 export const LITECOIN = 'Litecoin';
+export const DEFI = 'DeFi';
```

Comment:
I don't think this is deserving of its own category, recommend putting it in CRYPTO.  If we end up with so many markets like this that they deserve their own category we can create a new category at that time and move them all.

Reply:
(FWIW, I think a lot of the categories shouldn't be categories)

Reply:
This actually is in the "CRYPTO" category in the templates at the moment. It looks like CRYPTO has children already. Not sure how to stick the template in there when the second level is groupings anyway.